holoviews/plotting/bokeh/widgets.py

```diff
@@ -1,8 +1,20 @@
 import json
+from distutils.version import LooseVersion
+
 import param
+import bokeh
+from bokeh.io import push_notebook, Document
+
+if LooseVersion(bokeh.__version__) >= LooseVersion('0.11'):
+    old_bokeh = False
+    from bokeh.io import _CommsHandle
+    from bokeh.util.notebook import get_comms
```

Comment:
I know that handling multiple Comms at once is on their radar and might even land in 0.11.1. If need be I can mirror CommsHandle in HoloViews, it's a tiny class that's literally just meant to hold a few attributes.